### PR TITLE
Version v11.0.0

### PR DESCRIPTION
**Pull request by danjm**
📦 🚀

### CODE REVIEW

File: shared/notifications/index.js [hunk 1]
```diff
@@ -346,5 +367,31 @@ export const getTranslatedUINotifications = (t, locale) => {
           )
         : '',
     },
+    // This syntax is unusual, but very helpful here.  It's equivalent to `unnamedObject[NOTIFICATION_DROP_LEDGER_FIREFOX] =`
+    [NOTIFICATION_DROP_LEDGER_FIREFOX]: {
+      ...UI_NOTIFICATIONS[NOTIFICATION_DROP_LEDGER_FIREFOX],
+      title: t('notificationsDropLedgerFirefoxTitle'),
```
Comment (Gudahtt) [inline shared/notifications/index.js:373]: Looks like this announcement was brought in accidentally (the associated localized messages don't exist)

Comment (FrederikBolding) [reply shared/notifications/index.js]: Amended the commit and force pushed, should be the only commit that changed.

**Review by FrederikBolding — APPROVED** [review]
This RC has been QAed by engineers on the snaps team and is good to merge.

**Review by tmashuang — APPROVED** [review]

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**